sky/cli.py

```diff
@@ -3220,8 +3220,9 @@ def storage_delete(names: List[str], all: bool):  # pylint: disable=redefined-bu
     else:
         names = _get_glob_storages(names)
 
-    for name in names:
-        sky.storage_delete(name)
+    pool_obj = multiprocessing.Pool()
```

Comment:
Can we use `subprocess_utils.run_in_parallel` here?

Reply:
sure, `subprocess_utils.run_in_parallel` makes more sense.